.gitignore

```diff
@@ -1,5 +1,6 @@
 # IDE
 .idea
+*~
```

Comment:
what is this

Reply:
Bhai my IDE was generating multiple backup files(a settings enabled) with that signature(storageService.ts~) whenever I made a change in any file and when committing those files also included, So with this I'm ignoring those files.